### ISSUE TITLE
documentation

### ISSUE THREAD

**Issue by glyph** [chrivers/pyjaco#71]
Please include some documentation, or put some up on readthedocs or pyjaco.org.  It's easy to 'pip install' but a bit obscure to figure out what to do next.

**Comment by chrivers** [comment]
Absolutely! Would you like to help? We're very friendly, and we accept all the help we can get :-)
